pkg/cpuid/cpuid_arm64_unsafe.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build arm64
+
+package cpuid
+
+import (
+	"io/ioutil"
+	"unsafe"
+
+	"gvisor.dev/gvisor/pkg/log"
+)
+
+// The auxiliary vector of a process on the Linux system can be read from /proc/self/auxv,
+// and tags and values are stored as 8-bytes decimal key-value pairs on the 64-bit system.
```

Comment:
Minor nit: please wrap comments to 80 columns.

Reply:
Done. Thanks!